R/clean.R

```diff
@@ -95,10 +95,13 @@ find_meta_stopwords <- function(metadata,
 #' @param add_stopwords A vector of words to be added to the urbanism stopwords
 #' @return A vector of urbanism-related stopwords
 #' @export
-urbanism_stopwords <- function(add_stopwords = NULL) {
-  stop_words <-
-    c(add_stopwords, "city", "urban", "urbanism", "hab", "km")
+subject_stopwords <- function(add_stopwords = NULL, convert_to_regex = TRUE) {
+  stop_words <- c(add_stopwords, "city", "cities", "urban", "urbanism", "hab", "km",
+                  "emu", "european postgraduate master in urbanism", "advisor",
+                  "prof")
```

Comment:
I added a few more subject-specific words here.

Reply:
this is very specific: "european postgraduate master in urbanism". Maybe we want to keep it out of the function and add it in the argument `add_stopwords` , when calling it?

Reply:
Yes, I would even suggest we move out all subject-specific words to the paper, also to make them explicit to the reader. In this corpus, "emu" and "european postgraduate master of urbanism" might be as frequent as more general terms such as "city" and "urban".